CHANGELOG.md

```diff
@@ -4,10 +4,12 @@ As this project is pre 1.0, breaking changes may happen for minor version bumps.
 
 ## Pending
 
+## 0.43.0
```

Comment:
do you intend to get the https://github.com/stellar/java-stellar-sdk/pull/566/  into this release also?

Reply:
Yes!